cyraft/node.py

```diff
@@ -685,12 +753,22 @@ async def _on_term_timeout(self) -> None:
             self._term,
         )
         assert self._state == RaftState.LEADER, "Only leaders have a term timeout"
-        self._term += 1
```

Comment:
This change is already part of main, please make sure you have rebased correctly!